lib/fluent/config/element.rb

```diff
@@ -182,35 +182,45 @@ def secret_param?(key)
         false
       end
 
-      def param_type(key)
-        return nil if @corresponding_proxies.empty?
+      def options(key)
```

Comment:
How about `options_for(key)`(or other better name)? `options(key)` is a bit obscure for me.

Reply:
And if this method is used by this class only, could you make it a private method?

Reply:
Sorry, now I think that this method isn't needed anymore, I'll revert it.

This method is introduced for getting a default value:

```diff
+       def default_value(key)
+         options(key)[:default]
+       end
```

and unifying similar code in `param_type` method. But getting a default value from `options` is wrong as I mentioned in https://github.com/fluent/fluentd/pull/2685#discussion_r352413304.

I'll revert this code, and simply show `:default` for a default value since it can't determined in this context.

Reply:
> I'll revert this code, and simply show a text :default for a default value since it can't determined in this context.

It's still wrong.
A default value can be taken from `@corresponding_proxies`.
I've fixed it.